test/fixtures/es-modules/simple.wat

```diff
@@ -0,0 +1,18 @@
+;; Compiled using the WebAssembly Tootkit (https://github.com/WebAssembly/wabt)
+;; $ wat2wasm simple.wat -o simple.wasm
+
+(module
```

Comment:
just some readability stuff, feel free to ignore

```wat
;; Compiled using the WebAssembly Tootkit (https://github.com/WebAssembly/wabt)
;; $ wat2wasm simple.wat -o simple.wasm

(module
  (import "./wasm-dep.mjs" "jsFn" (func $jsFn (result i32)))

  (func $add (param $a i32) (param $b i32) (result i32)
    (i32.add (local.get $a) (local.get $b))
  )

  (func $addImported (param $a i32) (result i32)
    (i32.add (local.get $a) (call $jsFn))
  )

  (export "add" (func $add))
  (export "addImported" (func $addImported))
)
```

Reply:
I must admit I prefer the stack machine form these days, but named locals is always a good idea.

Reply:
I also prefer to define both imports and exports at the top of wasm modules for easier inspection.